frontend/src/components/pagedItemsTable/PagedItemsTable.tsx

```diff
@@ -169,27 +229,17 @@ function PagedItemsTable<T extends object>({
 		retrieveItems([...items])
 	}, [getDataObjectsByID, items])
 
-
-	// Return the ID that corresponds to the object displayed in a row of the table.
-	// We just find the object in the dataObjects map and return its corresponding
-	// key. This allows us to use data objects that don't have an explicit 'id' property.
-	function getRowKeyForDataObject(data: T) {
-		for (const key in tableDataState.objectMap) {
-			if (tableDataState.objectMap[key] === data) {
-				return key
-			}
-		}
-		return 'unknown-data-row-key'
-	}
-
 	return (
 		<>
 			{columns && (
 				<>
-					{usingFilters && pagedItems.filters && (
-						<FiltersBar filters={pagedItems.filters} clearFilters={clearFiltersCallback}></FiltersBar>
-					)}
-					<Table
+					<div>
+						{topBarExtra}
```

Comment:
It is a bit weird not to have the Link to Study button not vertically aligned with the Clear Filter button.

Reply:
I don't like frontend development.